assembly/utils/Bytes.ts

```diff
@@ -0,0 +1,46 @@
+export const enum BIT_LENGTH {
+    INT_8 = 1,
+    INT_16 = 2,
+    INT_32 = 4,
+    INT_64 = 8
+}
+
+export class Bytes {
+
+    static putUint16(b: u8[], v: u16): void {
```

Comment:
Let's make it more abstract -> provide the byteLength and based on that loop by shifting every time